### DISCUSSION TITLE
Is there a way to reinitialise quasar cli to use scss?

### DISCUSSION THREAD

**Discussion by elie464** [quasarframework/quasar#10290] [General - Components / Directives / etc] [answered]
I opted out of scss and noticed a lot of files like quasar.variables.scss isn't installed, any way i can get them after a project's been created? Cheers

**Comment by metalsadman** [comment, marked as answer]
@elie464 rename the css files to scss and change the reference in your `quasar.conf.js>css` to app.scss, you probably have to run quasar clean too, or delete .quasar, node_modules folder, and yarn.lock file and run `yarn`.

**Reply by elie464** [reply]
thanks
